src/Game.js

```diff
@@ -74,25 +74,42 @@ const playCard = (G, ctx, card) => {
     console.log(`${ctx.playerID} played`);
     const playerId = ctx.playerID;
     const playedCard = G.players[playerId].cards[card];
-    G.players[playerId].cards = G.players[playerId].cards.filter(c => c !== playedCard);
-    G.aboutToPlay[playerId] = null;
-
-    G.cardsPlayed.push(playedCard);
-    G.gameOver = isGameOver(G);
+    return {
+        ...G,
+        players: {
+            [playerId]: {
+                cards: G.players[playerId].cards.filter(c => c !== playedCard)
+            }
```

Comment:
this overwrites the entire contents of `players`...can you change it to something like this? 
```
            ...players
            [playerId]: {
                cards: G.players[playerId].cards.filter(c => c !== playedCard)
            }
```

Reply:
Sorry! Just fixed.